param.go

```diff
@@ -485,23 +533,85 @@ func newParamGroupedSlice(f reflect.StructField, c containerStore) (paramGrouped
 	return pg, nil
 }
 
-func (pt paramGroupedSlice) Build(c containerStore) (reflect.Value, error) {
-	var itemCount int
+func (pt paramGroupedSlice) getDecoratedValues(c containerStore) []reflect.Value {
+	var items []reflect.Value
+	stores := c.storesToRoot()
+	for _, c := range stores {
+		items = append(items, c.getDecoratedValueGroup(pt.Group, pt.Type.Elem())...)
+	}
+	return items
+}
+
+// searches the given container and its parent for matching group decorators
```

Comment:
:s/searches/search/g
:s/calls/call/g

possible suggestion: If any decorators return an error, that error is returned immediately. If all decorators succeed, nil is returned.